src/status_im2/common/bottom_sheet/view.cljs

```diff
@@ -247,9 +211,54 @@
                {:style (reanimated/apply-animations-to-style
                         {:opacity bg-opacity}
                         styles/backdrop)}]]
+             ;; VVVVVVV
+             ;; This is repeated because gesture-handler doesn't play nicely when it's conditionally
```

Comment:
It's hard for me to accept this @ibrkhalil. There must be a way to not simply copy and paste the whole thing. We should be able to extract the common logic into a separate function and remove the duplication.

Extracting to a separate function would have the added benefit of helping the bottom sheet component *not grow* any longer. This component is crying to be broken down into smaller chunks, really hard to read IMO.

Reply:
Much better!